python/hail/expr/types.py

```diff
@@ -663,20 +663,20 @@ def _select(self, *identifiers):
 
 class TVariant(Type):
     """
-    Hail type corresponding to :class:`hail.representation.Variant`.
+    Hail type corresponding to :class:`hail.genetics.Variant`.
```

Comment:
Yeah, I'd move this to genetics.types, etc.

Reply:
as above, OK if I do this in a directly following PR?